e2e/volume_ops_test.go

```diff
@@ -409,3 +412,41 @@ func testDisperse(t *testing.T) {
 	r.Nil(client.VolumeStop(disperseVolName), "disperse volume stop failed")
 	r.Nil(client.VolumeDelete(disperseVolName), "disperse volume delete failed")
 }
+
+func testEditVolume(t *testing.T) {
+	r := require.New(t)
+
+	var brickPaths []string
+
+	for i := 1; i <= 4; i++ {
+		brickPath, err := ioutil.TempDir(tmpDir, "brick")
+		r.Nil(err)
+		brickPaths = append(brickPaths, brickPath)
+	}
+
+	createReq := api.VolCreateReq{
+		Name: volname,
+		Subvols: []api.SubvolReq{
+			{
+				ReplicaCount: 2,
+				Type:         "replicate",
+				Bricks: []api.BrickReq{
+					{PeerID: gds[0].PeerID(), Path: brickPaths[0]},
+					{PeerID: gds[1].PeerID(), Path: brickPaths[1]},
+				},
+			},
+		},
+		Force: true,
+	}
+	_, err := client.VolumeCreate(createReq)
+
+	editMetadataReq := api.VolEditReq{
+		Metadata: map[string]string{
+			"owner": "gd2tests",
+		},
+	}
+	_, err = client.EditVolume(volname, editMetadataReq)
```

Comment:
You will have to check the response body. Also, test the replace key cases.

Reply:
Done. Please review.

Reply:
Added the check for the response as well as for the new volinfo generated after the changes.